README.md

```diff
@@ -11,15 +11,12 @@ For example:
 
 ## Features
 
-Currently implemented features include:
+Current implemented features include:
```

Comment:
Nitpick: this still reads as slightly off, maybe just "Implemented Features Include"

Reply:
will change